adminguide.rst

```diff
@@ -0,0 +1,514 @@
+
+By default, the queue is *enabled*, meaning that jobs can be submitted
+into the system. To disable job submission, e..g to prepare the system
+for a shutdown, use ``flux queue disable``. To restore queue access
+use ``flux queue enable``.
+
+~~~~~~~~~~~~~~~~~~~~~~~
+Stopping job allocation
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The queue may also be stopped with ``flux queue stop``, which disables
+further allocation requests from the job-manager to the scheduler. This
+allows jobs to be submitted, but stops new jobs from being scheduled.
+To restore scheduling use ``flux queue start``.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Flux queue idle and drain
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``flux queue drain`` and ``flux queue idle`` commands can be used
+to wait for the queue to enter a given state. This may be useful when
+preparing the system for a downtime.
+
+The queue is considered *drained* when there are no more active jobs.
+That is, all jobs have completed and there are no pending jobs.
+``flux queue drain`` is most useful when the queue is *disabled* .
+
+The queue is "idle" when there are no jobs in the RUN or CLEANUP state.
+In the *idle* state, jobs may still be pending. ``flux queue idle``
+is most useful when the queue is *stopped*.
+
+To query the current status of the queue use the ``flux queue status``
+command:
+
+.. code-block:: console
+
+ $ flux queue status -v
+ flux-queue: Job submission is enabled
+ flux-queue: Scheduling is enabled
+ flux-queue: 2 alloc requests queued
+ flux-queue: 1 alloc requests pending to scheduler
+ flux-queue: 0 free requests pending to scheduler
+ flux-queue: 4 running jobs
+
+
+.. _managing-jobs:
+
+^^^^^^^^^^^^^^^^^^
+Managing Flux Jobs
+^^^^^^^^^^^^^^^^^^
+    
+.. _expedite-jobs:
+
+~~~~~~~~~~~~~~~
+Expediting Jobs
+~~~~~~~~~~~~~~~
+
+.. _canceling-jobs:
+
+~~~~~~~~~~~~~~
+Canceling Jobs
+~~~~~~~~~~~~~~
+
+An active job may be canceled via the ``flux job cancel`` command. An
+instance owner may cancel any job, while a guest may only cancel their
+own jobs.
+
+All active jobs may be canceled with ``flux job cancelall``. By default
+this command will only print the number of jobs that would be canceled.
+To force cancellation of all matched jobs, the ``-f, --force`` option must
+be used:
+
+.. code-block:: console
+
+ $ flux job cancelall
+ flux-job: Command matched 5 jobs (-f to confirm)
+ $ flux job cancelall -f
+ flux-job: Canceled 5 jobs (0 errors)
+
+The set of jobs matched by the ``cancelall`` command may also be restricted
+via the ``-s, --states=STATES`` and ``-u, --user=USER`` options.
+
+
+.. _dedicated-application-time:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Dedicated Application Time
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _updating-flux:
+
+^^^^^^^^^^^^^^^^^^^^^^
+Updating Flux Software
+^^^^^^^^^^^^^^^^^^^^^^
+
+
+.. _troubleshooting:
+
+---------------
+Troubleshooting
+---------------
+
+
+.. _flux-logs:
+
+^^^^
+Logs
+^^^^
+
+.. _systemd-journal:
+
+~~~~~~~~~~~~~~~
+Systemd journal
+~~~~~~~~~~~~~~~
```

Comment:
Might mention `journalctl -u flux` as the way to access Flux log messages that the broker emits to stderr?